apple/RNSVGRenderable.mm

```diff
@@ -222,9 +223,30 @@ - (void)prepareForRecycle
 }
 #endif // RCT_NEW_ARCH_ENABLED
 
-UInt32 saturate(CGFloat value)
+static CGImageRef renderToImage(RNSVGRenderable *object, CGSize bounds, CGRect rect, CGRect *clip)
```

Comment:
```suggestion
static CGImageRef renderToImage(RNSVGRenderable *renderable, CGSize bounds, CGRect rect, CGRect *clip)
```

Or `element` or something like this would be better I think